app/controllers/request-to-go-live/choose-takes-payments-over-phone/post.controller.js

```diff
@@ -7,13 +7,13 @@ const goLiveStage = require('../../../models/go-live-stage')
 const formatServicePathsFor = require('../../../utils/format-service-paths-for')
 const { response } = require('../../../utils/response')
 const { requestToGoLive } = require('../../../paths').service
-const { validPaths, ServiceUpdateRequest } = require('../../../models/ServiceUpdateRequest.class')
+const { ServiceUpdateRequest } = require('../../../models/ServiceUpdateRequest.class')
```

Comment:
Is it possible to use `@models/ServiceUpdateRequest.class` here?

Reply:
Probably yes, I'll do that